.idea/.name

```diff
@@ -0,0 +1 @@
+Tutorial
```

Comment:
It's not needed, in Android Studio 2.1 if the .idea files are missing, the gradle task will handle it and create a local ones, but there will be an error message right after importing files to the Studio.